README.rst

```diff
@@ -98,9 +98,14 @@ is not (yet) supported:
 .. _citeproc-js: https://github.com/juris-m/citeproc-js
 
 
-Running the Tests
+Local Install and Running the Tests
 -----------------
 
-First clone the `citeproc-test suite`_ so that it sits next to the
-citeproc-py directory. Now you can run ``citeproc-test.py`` (in the ``tests``
-directory). Run ``citeproc-test.py --help`` to see its usage information.
+First clone the `citeproc-py`_ repository and install the submodules with `git
+submodule init`_ and `git submidule update`_. Then install with `python setup.py
+install`_. Then move to the ``tests`` directory and run ``python citeproc-test.py``. 
```

Comment:
I don't think that `python setup.py install` is currently recommended practice... `pip install .` AFAIK would be better for installation and `pip install -e .` for "installation for development"

Reply:
Yup, good catch. I've switched it.